bin/compile

```diff
@@ -1,8 +1,20 @@
 #!/usr/bin/env bash
 
 export BUILD_DIR=$1
+export ENV_DIR=$3
 export POETRY_VERSION=1.0.10
 
+# Extract config vars into the environment.
+# From https://devcenter.heroku.com/articles/buildpack-api
+denylist_regex='^(PATH|GIT_DIR|CPATH|CPPATH|LD_PRELOAD|LIBRARY_PATH)$'
+if [ -d "$ENV_DIR" ]; then
+  for e in $(ls $ENV_DIR); do
+    echo "$e" | grep -qvE "$denylist_regex" &&
+    export "$e=$(cat $ENV_DIR/$e)"
+    :
+  done
+fi
+
```

Comment:
I would have picked it out individually, just as you initially did, because God knows what clashes there could happen - we use variables for configuration very extensively...

```suggestion
export DISABLE_POETRY_CREATE_RUNTIME_FILE="$(cat $ENV_DIR/DISABLE_POETRY_CREATE_RUNTIME_FILE 2>/dev/null || true)"

```

Reply:
Yeah definitely, that makes sense, I'll make the change.

Reply:
Okay, done.